symplyphysics/conditions/electricity/maxwell_equations/faradays_law_of_induction.py

```diff
@@ -0,0 +1,64 @@
+from sympy import Eq, Derivative
+from sympy.core import Equality
+from symplyphysics import (units, Quantity, Symbol, validate_input)
+from symplyphysics.core.fields.operators import curl_operator as rotor
+from symplyphysics.core.fields.vector_field import VectorField
+from symplyphysics.core.dimensions import assert_equivalent_dimension
+
+## Description
+## Faraday's law of induction states that a change in magnetic induction generates a vortex electric field.
+
+## Law is: rot(E(x, y, z, t)) = -d(B(x, y, z, t)) / dt, where
+## E - the vector of electric intensity,
+## B - the vector of magnetic induction,
+## x, y, z - coordinates in a rectangular coordinate system,
+## t - tiime,
+## rotor - rotor of the vector of magnetic induction,
+## d / dt - time derivative.
+
+x_coor = Symbol('x_coor', units.length)
+y_coor = Symbol('y_coor', units.length)
+z_coor = Symbol('z_coor', units.length)
+time = Symbol('time', units.time)
+
+
+def faradays_law_of_induction(electric_intensity_: VectorField,
```

Comment:
It looks like condition. But I'm not sure, condition of what. Could you clarify, when this equation is valid and rename it accordingly.